spec/poros/food_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe Food do
+  it "exists and has attributes" do
+    data = {description: "Food Test Description"}
```

Comment:
Make sure to include all the data you need in the test data, not just the description field